### PR TITLE
more content for the class tomorrow

### PR DESCRIPTION
**Pull request by keponk**
added text description for each of the setup items plus simplified the "things to do". Link to homework is better not (more simple) but was missing the `/lessons/` part of the url.

MOST IMPORTANT: I know it is a last minute change and I apologize for not bringing this up earlier. I really really really would like to change codepen for glitch.com , it really is cool and i think a better tool right as of 2020, also for new devs. I had completely forgotten about it but I really would prefer it. Let me know if that's cool.

### CODE REVIEW

File: lessons/1_setup/index.html [hunk 1]
```diff
@@ -145,43 +145,103 @@ <h3>What could be a Final Project?</h3>
...
                 >https://code.visualstudio.com/download</a
-              ><small>* ask us about the recommended plugins</small>
+              ><small
+                >Recommended plugins
+                <ol>
+                  <li>bäumer.vscode-eslint</li>
+                  <li>esbenp.prettier-vscode</li>
+                  <li>bäumer.vscode-eslint</li>
+                  <li>ms-vsliveshare.vsliveshare</li>
+                  <li>ms-vsliveshare.vsliveshare-pack</li>
```
Comment (MrTim) [inline lessons/1_setup/index.html:180]: PKief.material-icon-theme that one is good as well, adds nice icons

**Review by MrTim — APPROVED** [review]
All good for me

File: lessons/1_setup/index.html (hunk 1 above)
Comment (keponk) [reply lessons/1_setup/index.html:180]: oh that's nice i didn't know that one. Maybe we can use this as a chance to give a small tutorial to the students on how to set it up nicely :)

Comment (MrTim) [reply lessons/1_setup/index.html:180]: yep, sure :)

Comment (keponk) [reply lessons/1_setup/index.html:180]: done

**Pull request merged**